assets/javascript/script.js

```diff
@@ -82,32 +102,10 @@ btnHboEl.addEventListener("click", function(event){
 
         /*populate movie variables with api data
 
-
    */ 
 // }
 
-function getMotNAPI() {
-    const options = {
-        method: 'GET',
-        headers: {
-            'X-RapidAPI-Key': '54664c79a4msh5e441cee8ee88b1p10830cjsn8417ae33858b',
-            'X-RapidAPI-Host': 'streaming-availability.p.rapidapi.com'
-        }
-    };
-    
-    fetch('https://streaming-availability.p.rapidapi.com/search/basic?country=us&service='+ streamingService +'&type=movie&genre=27&page=1&output_language=en&language=en', options)
-    // create empty string for all streaming services so
-    // add click event that listens for each streaming service and pulls all horror movies from those services
-        .then(response => response.json())
-        .then(response => console.log(response))
-        .catch(err => console.error(err));
-}
-
-smashedBtnEl.addEventListener("click", function(event){
```

Comment:
We need to leave 106-110. That’s the event listener for the get smashed button.

Reply:
okay! It looks like i moves it up with the other stuff. i'll go ahead and move it back down.